src/Storage/Storage/Storage.csproj

```diff
@@ -13,10 +13,10 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Data.Tables" Version="12.8.0" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
-    <PackageReference Include="Azure.Storage.Files.DataLake" Version="12.14.0" />
-    <PackageReference Include="Azure.Storage.Files.Shares" Version="12.14.0" />
-    <PackageReference Include="Azure.Storage.Queues" Version="12.14.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.17.0" />
```

Comment:
The SDK version update also need happen in Storage.Management.Test

Reply:
Done